src/core/selinux-access.h

```diff
@@ -6,15 +6,37 @@
 #include "bus-util.h"
 #include "manager.h"
 
-int mac_selinux_generic_access_check(sd_bus_message *message, const char *path, const char *permission, sd_bus_error *error);
+#define MAC_SELINUX_CLASS_SYSTEM            "system"
+#define MAC_SELINUX_CLASS_UNIT              "service"
+
+#define MAC_SELINUX_UNIT_PERM_ADDDEPENDENCY "modify"
+#define MAC_SELINUX_UNIT_PERM_DISABLE       "disable"
+#define MAC_SELINUX_UNIT_PERM_ENABLE        "enable"
+#define MAC_SELINUX_UNIT_PERM_LINK          "enable"
+#define MAC_SELINUX_UNIT_PERM_MASK          "disable"
+#define MAC_SELINUX_UNIT_PERM_PRESET        "modify"
+#define MAC_SELINUX_UNIT_PERM_REENABLE      "enable"
+#define MAC_SELINUX_UNIT_PERM_REVERT        "modify"
+#define MAC_SELINUX_UNIT_PERM_STATUS        "status"
+#define MAC_SELINUX_UNIT_PERM_UNMASK        "enable"
```

Comment:
hmm, why have these defines? Isn't it easier to read to just use the literal strings?

Reply:
so, the issue here is that this changes the permission names around from the status quo ante. I'd prefer if we could initially just reuse the old verbs (simply because the existing selinux policy probably already caters for it), and then in a separate commit move these to new verbs. And these new verbs need to be added with the knowledge and OK of the folks maintaining the policy. I want some blessing from them before this can go in, given this means bigger changes in the policy being necessary

Reply:
reworked